cluster/k8s-1.9.3/provider.sh

```diff
@@ -42,8 +44,21 @@ EOF
 }
 
 function build() {
+    # Build everyting and publish it
     ${KUBEVIRT_PATH}hack/dockerized "DOCKER_TAG=${DOCKER_TAG} PROVIDER=${PROVIDER} ./hack/build-manifests.sh"
     make build docker publish
+
+    # Make sure that all nodes use the newest images
+    container=""
+    for arg in ${docker_images}; do
+        local name=$(basename $arg)
+        container="${container} ${manifest_docker_prefix}/${name}:${docker_tag}"
+    done
+    local num_nodes=${VAGRANT_NUM_NODES-0}
+    num_nodes=$((num_nodes + 1))
+    for i in $(seq 1 ${num_nodes}); do
```

Comment:
did you mean for this to be a nested loop?!

Reply:
I append all containers in the loop before https://github.com/kubevirt/kubevirt/pull/761/files#diff-e57e3ad015ee4f324d07c1d2d4b4d738R55, then let xargs do its work on every node.

Reply:
i see now